.drone.yml

```diff
@@ -10,18 +10,18 @@ trigger:
 
 steps:
   - name: checkout
-    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.1.0
     commands:
       - git fetch --tags
       - git checkout ${DRONE_TAG}
       - git submodule update --init --recursive
 
   - &deploy
     name: cs-dev
-    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.1.0
     environment:
       CHART_NAME: hocs-toolbox
-      CHART_VERSION: 1.0.0
+      CHART_VERSION: ^2.0.0
```

Comment:
https://github.com/UKHomeOffice/hocs-helm-charts/blob/main/charts/hocs-toolbox/Chart.yaml
no good spot though, most are because they're being developed at the same time at the moment, but should now start to diverge.